kobo/worker/__init__.py

```diff
@@ -2,10 +2,7 @@
 
 
 from __future__ import absolute_import
-import os
-
-from .task import *
```

Comment:
This is causing a crash in one of kobo's own tasks which in turn causes basic top-level imports to fail:

````
python -c 'import kobo.worker'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python2.6/site-packages/kobo/worker/__init__.py", line 10, in <module>
    TaskContainer.register_module(tasks)
  File "/usr/lib/python2.6/site-packages/kobo/plugins.py", line 173, in register_module
    __import__(module.__name__, {}, {}, module_list)
  File "/usr/lib/python2.6/site-packages/kobo/worker/tasks/task_shutdown_worker.py", line 4, in <module>
    from kobo.worker import TaskBase
ImportError: cannot import name TaskBase
````

At the very least this has to be fixed within kobo itself, but it could well break other code outside of kobo as well since it's a source-incompatible change to stop exporting these names here.

Reply:
Thank you for your review! I'll add `from .task import *` back.